nens_auth_client/backends.py

```diff
@@ -96,8 +97,14 @@ def authenticate(self, request, claims):
         if username is None:
             return
 
+        email = claims.get("email")
+
         try:
-            user = UserModel.objects.get(username__iexact=username, remote=None)
+            user = UserModel.objects.get(
+                Q(username__iexact=username),
+                Q(remote__isnull=True)
+                | (Q(remote__isnull=False) & Q(email__iexact=email)),
+            )
```

Comment:
Ik vroeg me even af of het met de volgorde van OR en AND goed ging. Maar AND heeft inderdaad een hogere prio dan OR.

Reply:
In mijn code heb ik haakjes gebruikt om het duidelijker te maken.

Reply:
Glad overheen gekeken, die haakjes. Zucht :-)